docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,309 @@
+  desktop shortcuts
+- verify that the *Application* and *Monitor Servers* are up
+- :ref:`contact us <contact_us>` for assistance.
+
+.. _choose_migration_path:
+
+Choose Migration Path
+---------------------
+
+If your instance is already using v3 onion services, choose our
+:ref:`Standard Migration Procedure <migration_standard>`.
+
+Instances that have not yet enabled v3 
+:ref:`onion services <glossary_onion_service>` should choose the 
+:ref:`Alternate Migration Procedure <migration_alternate>`. 
+
+
+.. _prune_submissions:
+
+Delete Old Submissions from the Server
+--------------------------------------
+
+In coordination with journalists, ensure that any old or unneeded
+submissions have been deleted from the server. Pruning old submissions
+will reduce the size and improve the speed of your server backup. 
+Journalists can delete unneeded submissions via the *Journalist Interface.*
+
+
+.. _download_focal:
+
+Download and Verify Ubuntu 20.04 LTS (Focal) Installation Media
+----------------------------------------------------------------
+
+Follow our instructions to 
+:doc:`download and verify Ubuntu Server 20.04 LTS <../servers>` and
+install the .iso file onto a USB stick.
+
+You have now completed all the preparatory steps. The rest of the 
+migration procedure will be completed during your maintenance window. 
+
+Migration
+~~~~~~~~~ 
+
+.. _migration_standard:
+
+Standard Migration Procedure
+----------------------------
+Perform these steps if your instance is already using v3 onion services. Ensure
+you have completed the :ref:`preparatory steps <focal_prep>`.
+
+#. Announce your maintenance window. As part of this procedure, your servers
+   will become unreachable.
+#. :doc:`Take a backup of the current instance <../backup_and_restore>`. Once 
+   you have taken a backup of the servers, power them off.
+
+    .. warning::
+
+     The next steps will overwrite existing data on the servers.
+
+     Data from the *Monitor Server* will not be restored after the backup.
+     If you require historical data from the *Monitor Server*, archive it
+     separately before proceeding.
+#. Follow the instructions on 
+   :ref:`hardware migration for instances using v2+v3 or v3 onion services <migrate_v3>`. 
+
+   You will be instructed to install Ubuntu 20.04, overwriting the 
+   existing operating system on the servers,
+   and you will be guided through reinstalling SecureDrop.
+
+   Once you have reinstalled SecureDrop, you will be instructed on how to
+   apply your backup file, restoring data and Tor credentials.  
+#. Ensure that *Journalist* and *Admin Workstations* are up-to-date and can
+   access the *Source* and *Journalist Interfaces*. When your instance 
+   enabled v3 onion services, one of the important steps was to ensure that
+   all Journalist and Admin Workstations were 
+   :ref:`updated with the new Tor credentials <update_tails_v3>`. If you 
+   have not yet completed this step, you will need to do so now.
```

Comment:
I would simplify this a bit, e.g.:

> Any v2 onion services will be disabled at this point. If you have not yet updated the onion service configurations for all Journalist and Admin Workstations, you must do so now to preserve access.

(Replacing the paragraph that begins with "When..")